contracts/AssetPool.sol

```diff
@@ -66,6 +70,12 @@ contract AssetPool is Ownable {
         uint256 timestamp
     );
     event WithdrawalTimedOut(address indexed underwriter, uint256 timestamp);
+    event UpgradeToNewAssetPoolCompleted(
+        address indexed underwriter,
+        uint256 collateralAmount,
+        uint256 covAmount,
```

Comment:
I think this should be the COV amount from this pool. This is what `WithdrawalInitiated` includes and this is the event from the current contract, so should inform about the current contract's state changes.

Reply:
The reason behind it was that this contract was transferring new underwriter tokens to underwriters, but since the latest approach is that a new asset pool will handle transfer of new cov tokens we can change it